src/Microsoft.Azure.SignalR/HubHost/TimerAwaitable.cs

```diff
@@ -1,109 +1,109 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
What changed here? Just the license?

Reply:
Just the license. No idea why the whole file is marked as changed.